complete higher lower game

mercurrie opened the pull request:
created art.py: contains various ASCII artwork
created game_data.py: contains data to be used in main.py
created main.py: contains vital functions to complete program
included more information to README.md

mercurrie committed:
complete higher lower game

created art.py
created game_data.py
created main.py
included more information to README.md

michaelhousman commented on main.py:
```diff
@@ -0,0 +1,55 @@
+        solution = 'a'
+    elif choice_b['follower_count'] > choice_a['follower_count']:
+        solution = 'b'
+    else:
```
Not sure what this `else` is for.  Does this handle the scenario where the follower count is equal?

mercurrie replied:
That is correct. The else is to handle the case the follower counts are equal

michaelhousman commented on main.py:
```diff
@@ -0,0 +1,55 @@
+def compare(choice_a: dict, choice_b: dict) -> bool:
+    user_answer = input('Who has more instagram followers? Type "A" or "B": ').lower()
+    
+    if choice_a['follower_count'] > choice_b['follower_count']:
```
See if you can do this in one line:

solution = 'a' if <condition> else 'b'

mercurrie replied:
recommended changes made

michaelhousman replied:
Great

michaelhousman commented on main.py:
```diff
@@ -0,0 +1,55 @@
+    else:
+        solution = user_answer
+        
+    if user_answer == solution:
```
One line to replace these 4:

return user_answer == solution

Try it.

mercurrie replied:
Ah I understand

michaelhousman commented on main.py:
```diff
@@ -0,0 +1,55 @@
+        print(f"You're right! Current score: {score}.")
+        subject1 = subject2
+        subject2 = random.choice(data)
+        print_UI(subject1 = subject1, subject2 = subject2)
```
Don't mix snake_case and capital letters

mercurrie replied:
Mental note taken. will only code lowercase snake_case

michaelhousman replied:
Cool

michaelhousman commented on game_data.py:
```diff
@@ -0,0 +1,302 @@
+data = [
```
This is good.   For practice, try converting this to JSON and importing it in the Python script.

mercurrie replied:
I converted 'game_data.py' into 'game_data.json', imported the file in 'main.py', opened the file, loaded it to a variable, and closed the file at the end of the program

michaelhousman replied:
Excellent

michaelhousman reviewed (changes requested):
This is good.  Left you a few things to work on.

mercurrie committed:
Update main.py

changed print_ui() to snake case,
made quality of life changes to compare(),

mercurrie committed:
changes to main.py and game_data.py

converted 'game_data.py' into json file.

imported and loaded 'game_data.json into main.py

mercurrie committed:
Update main.py

close json file in main.py

michaelhousman reviewed (approved):
This one looks good.  You can go ahead and merge :)
